lang/fr_FR.json

```
@@ -0,0 +1,416 @@
+    "Page": "Page",
+    "Page <1></1> of {{totalPages, number}}": "Page <1></1> sur {{totalPages, number}}",
+    "Passwords must match.": "Les mots de passe doivent correspondre.",
+    "Playable": "Jouable",
+    "Players": "Joueurs",
+    "Please include as much evidence as possible to support your claim. This could include screenshots, videos, links to suspicious profiles, or any other relevant information that demonstrates the alleged violation. Describe each piece of evidence in detail, explaining why it suggests a violation of the rules. The more comprehensive and clear your submission, the more efficiently we can evaluate and address the issue.": "Veuillez inclure autant de preuves que possible pour étayer votre réclamation. Cela peut inclure des captures d'écran, des vidéos, des liens vers des profils suspects ou toute autre information pertinente démontrant la violation présumée. Décrivez chaque élément de preuve en détail, en expliquant pourquoi il suggère une violation des règles. Plus votre soumission est complète et claire, plus nous pourrons évaluer et résoudre efficacement le problème.",
+    "Points": "Points",
+    "Points Earned Since {{date}}": "Points gagnés depuis le {{date}}",
+    "Preferences": "Préférences",
+    "Previous": "Précédent",
+    "Previous {{count, number}}": "Précédent {{count, number}}",
+    "Privacy Policy": "Politique de confidentialité",
+    "Profile": "Profil",
+    "Progress": "Progression",
+    "Progress was reset successfully.": "La progression a été réinitialisée avec succès.",
+    "Proposing a new article idea.": "Proposer une nouvelle idée d'article.",
+    "Proposing art updates.": "Proposer des mises à jour artistiques.",
+    "Quality Assurance": "Assurance qualité",
+    "Questions about art-related rule changes.": "Questions sur les changements de règles liés à l'art.",
+    "RAEvents": "RAEvents",
+    "RANews": "RANews",
+    "RSS Feeds": "Flux RSS",
+    "Rarity": "Rareté",
+    "Recent Forum Posts": "Messages récents du forum",
+    "Recent Posts": "Messages récents",
+    "Registered Players": "Joueurs enregistrés",
+    "Release Date": "Date de sortie",
+    "Remove from Want to Play Games": "Retirer des jeux à jouer",
+    "Removed {{gameTitle}} from playlist!": "{{gameTitle}} retiré de la liste !",
+    "Removed {{gameTitle}}!": "{{gameTitle}} retiré !",
+    "Removing...": "Retrait...",
+    "Reorder Site Awards": "Réorganiser les récompenses du site",
+    "Report Issue": "Signaler un problème",
+    "Report Issue - {{achievementTitle}}": "Signaler un problème - {{achievementTitle}}",
+    "Report to DevCompliance": "Signaler à DevCompliance",
+    "Report to QATeam": "Signaler à l'équipe QA",
```

La traduction devrait rester "QATeam" puisqu'il s'agit d'un compte
```suggestion
    "Report to QATeam": "Signaler à QATeam",
```